Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.xaml

```diff
@@ -65,11 +71,6 @@
                                 </VisualState>
                                 <VisualState x:Name="Loaded">
                                     <Storyboard>
-                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="Progress"
```

Comment:
Not sure why this is removed? And should the visibility be changed to collapsed here as well?

Reply:
@nmetulev if you do not specify distinct values it reverts to default values set. As you can see the property was being set in Loaded but not in Failed. They should both behave in similar fashion.
I have tested it and the progress does stop on Loaded or Failed

Reply:
Ah, makes sense